dali/test/python/test_eager_cpu_only.py

```diff
@@ -214,6 +305,12 @@ def test_jpeg_compression_distortion_cpu():
     check_single_input('jpeg_compression_distortion', quality=10)
 
 
+def test_image_decoder_crop_device():
```

Comment:
What is this name supposed to mean? Normally "device" means either "GPU" or, in DALI terms, a configurable device (cpu/gpu) - this can be neither in CPU-only tests.

Reply:
Names are taken from `test_dali_cpu_only.py`, seemed like a good idea for them to be the same.